climada/hazard/tc_tracks_synth.py

```diff
@@ -654,6 +662,20 @@ def _decay_calc_coeff(x_val, v_lf, p_lf):
                          climada.hazard.tc_tracks.CAT_NAMES[close_scale - 2])
             v_rel[ss_scale] = v_rel[close_scale]
             p_rel[ss_scale] = p_rel[close_scale]
+        elif v_rel[ss_scale] < 0:
+            LOGGER.error('Landfall wind speed decay parameter is negative '
+                         'for category %s. Please check or use a larger '
+                         'number of events to obtain more sensible landfall '
+                         'decay values for wind speed.',
+                         climada.hazard.tc_tracks.CAT_NAMES[ss_scale - 2])
+            raise ValueError
+        elif p_rel[ss_scale][0] < 0 or p_rel[ss_scale][1] < 0:
+            LOGGER.error('Landfall pressure decay parameter is negative '
+                         'for category %s. Please check or use a larger '
+                         'number of events to obtain more sensible landfall '
+                         'decay values for pressure.',
+                         climada.hazard.tc_tracks.CAT_NAMES[ss_scale - 2])
+            raise ValueError
```

Comment:
This is to check that landfall parameters imply a decay of TC intensity. If they do not (i.e. they would enhance the TC strength), it issues an error.

Reply:
I think this is not really a good solution. Currently, the tc_track_synth easily throws errors and the user does not necessarily know what to do with it. Hence, it is better to write the code with extra assumptions...  But maybe first, could you please explain why a larger number of tracks would help?

Reply:
It is extremely unlikely that these parameters are negative. If they are, it can only be because you used a very low number of historical tracks as input and these by very very very bad luck increase intensity at landfall. The suggestion for improvements I suggested above (but which are in my view out of scope for this PR) would also avoid this issue. Hence in the meantime the proposition to raise an error here suggesting to use more historical tracks - should I change this to a warning?

Reply:
I agree however that the tc_track_synth throws too many error, which could be improved in a future PR.

Reply:
I reformulated your error message to be clearer for the user. Please note that for errors one should not use the Logger.

Reply:
Question: Can you please provide a citation for the 'intensity at landfall cannot increase' and include it in the error message?

Reply:
Idem for the 'landfall pressure cannot increase at landfall'?